packages/core/watchPromise/index.md

```diff
@@ -0,0 +1,25 @@
+---
+category: Reactivity
+---
+
+# watchPromise
+
+Watch with saving async functions call order.
+
+## Usage
```

Comment:
Can you give a more clear example of how this is supposed to work? It is not very clear from this example. `asyncFn` isn't ever used, `cleanQueue` isn't ever given an example of being used. What is current vs source?

Reply:
> Can you give a more clear example of how this is supposed to work? It is not very clear from this example. `asyncFn` isn't ever used, `cleanQueue` isn't ever given an example of being used. What is current vs source?

thank you for you review. My docs was really awful. Please check it. Is that better now ? I thought about making a small demo but I found it useless. Do you have any idea for a good demo for this function ?